automation/test.sh

```diff
@@ -108,6 +108,7 @@ elif [[ $TARGET =~ wg-s390x ]]; then
     export KUBEVIRT_PROVIDER=${TARGET/-wg-s390x}
 elif [[ $TARGET =~ wg-arm64 ]]; then
     export KUBEVIRT_PROVIDER=${TARGET/-wg-arm64}
+    export KUBEVIRT_COLLECT_CONTAINER_RUNTIME_DEBUG=true
```

Comment:
that env var would probably be worth an addition to the docs, or in the README.md - not sure where to put this from the top of my head though.